examples/configGeneration.md

```diff
@@ -4,17 +4,19 @@
 
 ## ConfigMap generation and rolling updates
 
-Kustomize provides two ways of adding ConfigMap in one `kustomization`, either by declaring ConfigMap as a [resource] or declaring ConfigMap from a ConfigMapGenerator. The formats inside `kustomization.yaml` are 
+Kustomize provides two ways of adding ConfigMap in one `kustomization`, either by declaring ConfigMap as a [resource] or declaring ConfigMap from a ConfigMapGenerator. The formats inside `kustomization.yaml` are
```

Comment:
Kustomize can accept a pre-declared ConfigMap as just another [resource] from its resource list, or it can _generate_ a ConfigMap, filling it with data from various sources.